### PR TITLE
docs: update README to be auto-included in openfeature.dev

### PR DESCRIPTION
**Pull request by toddbaert**
Updating README to conform to auto-inclusion for openfeature.dev

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,42 +1,49 @@
...
+  <!-- x-release-please-start-version -->
+
+  <a href="https://github.com/open-feature/java-sdk/releases/tag/v1.4.3">
+    <img alt="Release" src="https://img.shields.io/static/v1?label=release&message=v1.4.3&color=blue&style=for-the-badge" />
+  </a>  
+
+  <!-- x-release-please-end -->
+  <br/>
+  <a href="https://bestpractices.coreinfrastructure.org/projects/6601">
+    <img alt="CII Best Practices" src="https://bestpractices.coreinfrastructure.org/projects/6601/badge" />
```
Comment (beeme1mr) [inline README.md:28]:
```suggestion
  <a href="https://bestpractices.coreinfrastructure.org/projects/6241">
    <img alt="CII Best Practices" src="https://bestpractices.coreinfrastructure.org/projects/6241/badge" />
```

File: README.md [hunk 2]
```diff
@@ -1,42 +1,58 @@
...
+  <a href="https://github.com/open-feature/java-sdk/releases/tag/v1.4.3">
+    <img alt="Release" src="https://img.shields.io/static/v1?label=release&message=v1.4.3&color=blue&style=for-the-badge" />
+  </a>  
+
+  <!-- x-release-please-end -->
+  <br/>
+  <a href="https://javadoc.io/doc/dev.openfeature/sdk">
+    <img alt="Javadoc" src="https://javadoc.io/badge2/dev.openfeature/sdk/javadoc.svg" />
+  </a>
+  <a href="https://javadoc.io/doc/dev.openfeature/sdk">
```
Comment (beeme1mr) [inline README.md:30]:
```suggestion
  <a href="https://maven-badges.herokuapp.com/maven-central/dev.openfeature/sdk">
```

**Review by beeme1mr — APPROVED** [review]
Looks good! Since you're referencing the in-memory provider, we should wait until the next release is ready to be merged.

https://github.com/open-feature/java-sdk/pull/558

File: README.md [hunk 3]
```diff
@@ -72,68 +89,153 @@ dependencies {
...
+    OpenFeatureAPI api = OpenFeatureAPI.getInstance();
+    api.addHooks(new ExampleHook());
+    
+    // add a hook on this client, to run on all evaluations made by this client
+    Client client = api.getClient();        
+    client.addHooks(new ExampleHook());
+    
+    // add a hook for this evaluation only
+    Boolean retval = client.getBooleanValue(flagKey, false, null,
+            FlagEvaluationOptions.builder().hook(new ExampleHook()).build());
```
Comment (Kavindu-Dodan) [inline README.md:201]: To align with all other examples, 

```suggestion
```java
  // add a hook globally, to run on all evaluations
  OpenFeatureAPI api = OpenFeatureAPI.getInstance();
  api.addHooks(new ExampleHook());
  
  // add a hook on this client, to run on all evaluations made by this client
  Client client = api.getClient();        
  client.addHooks(new ExampleHook());
  
  // add a hook for this evaluation only
  Boolean retval = client.getBooleanValue(flagKey, false, null,
          FlagEvaluationOptions.builder().hook(new ExampleHook()).build());
```

**Review by Kavindu-Dodan — APPROVED** [review]
Left two comments, but the rest looks solid

File: README.md [hunk 4]
```diff
@@ -144,60 +246,99 @@ OpenFeatureAPI.getInstance().onProviderStale((EventDetails eventDetails) -> {
...
-To develop a provider, you need to create a new project and include the OpenFeature SDK as a dependency.
-This can be a new repository or included in [the existing contrib repository](https://github.com/open-feature/java-sdk-contrib) available under the OpenFeature organization.
-Finally, you’ll then need to write the provider itself.
-This can be accomplished by implementing the `FeatureProvider` interface exported by the OpenFeature SDK.
+If you'd like your provider to support firing events, such as events for when flags are changed in the flag management system, extend `EventProvider`.
 
 ```java
-public class MyProvider implements FeatureProvider {
-@Override
+class MyEventProvider extends EventProvider {
```
[the remark this answers is not in the document]
Comment (toddbaert) [reply README.md:319]: I was debating that. If you think it's a good idea too, I'll make that change.

**Pull request merged**